frontend/src/app/main/data/workspace/libraries_helpers.cljs

```diff
@@ -316,19 +316,21 @@
             :library (pretty-file library-id state))
 
   (let [file          (wsh/get-file state file-id)
-        components-v2 (get-in file [:options :components-v2])]
-    (loop [pages (vals (get file :pages-index))
+        components-v2 (get-in file [:options :components-v2])
+        deleted-components (ctkl/deleted-components-seq file)]
+    ;; We want to update the items inside the :objects of the deleted components. Those aren't in any page
+    (loop [items (concat (vals (get file :pages-index)) deleted-components)
            changes (pcb/empty-changes it)]
-      (if-let [page (first pages)]
-        (recur (next pages)
+      (if-let [item (first items)]
```

Comment:
I'd name this `container`. because this concept designs specifically this: an object that may be either a page or a component, and that contains `:objects`.